stable/yugabyte/templates/service.yaml

```diff
@@ -405,6 +405,7 @@ spec:
           - "/bin/bash"
           - "-c"
           - |
+          {{- $preCommands := (ternary $root.Values.master.advanced.preCommands $root.Values.tserver.advanced.preCommands (eq .name "yb-masters")) }} {{- with $preCommands }} {{ . | nindent 12 }} {{- end }}
```

Comment:
Nit: most of our charts use
`true | <branch true> <branch false>` syntax for ternary.

Reply:
Fixed :)